tests/foreman/ui/test_contenthost.py

```diff
@@ -120,7 +150,14 @@ def run_remote_command_on_content_host(command, vm_module_streams):
     indirect=True,
 )
 @pytest.mark.no_containers
-def test_positive_end_to_end(session, default_location, module_repos_collection_with_manifest, vm):
+def test_positive_end_to_end(
+    vm,
+    session,
+    module_org,
+    setting_update,
+    default_location,
+    module_repos_collection_with_manifest,
+):
```

Comment:
Can you please:
1) Make the first paragraph of a docstring more high-level, describe in one simple sentence what the test tests, and
2) Add `:steps:` where you describe it more in detail?